private-templates-service/azuredeploy.json

```diff
@@ -2,145 +2,149 @@
   "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
-      "sites_adaptivecms_name": {
-          "defaultValue": "adaptivecmsdeploy",
-          "type": "String"
-      },
-      "mongodb_connection_string": {
-        "type": "String"
-      },
-      "app_id": {
-        "type": "String"
-      },
-      "location": {
-        "defaultValue": "West US",
-        "type": "String"
-      },
-      "server_farm_id": {
-        "type": "String"
-      },
-      "docker_registry_server_uri": {
-        "defaultValue": "https://acmsregistry.azurecr.io",
-        "type": "String"
-      },
-      "docker_registry_server_username": {
-        "defaultValue": "ACMSRegistry",
-        "type": "String"
-      },
-      "docker_registry_server_password": {
-        "type": "String"
-      },
-      "telemetry_opt_in": {
-          "type": "string",
-          "allowedValues": [
-              "Yes",
-              "No"
-          ]
-      },
+    "sites_adaptivecms_name": {
+      "defaultValue": "adaptivecmsdeploy",
+      "type": "String"
+    },
+    "mongodb_connection_string": {
+      "type": "String"
+    },
+    "app_id": {
+      "type": "String"
+    },
+    "location": {
+      "defaultValue": "West US",
+      "type": "String"
+    },
+    "server_farm_id": {
+      "type": "String"
+    },
+    "docker_registry_server_uri": {
+      "defaultValue": "https://acmsregistry.azurecr.io",
+      "type": "String"
+    },
+    "docker_registry_server_username": {
+      "defaultValue": "ACMSRegistry",
+      "type": "String"
+    },
+    "docker_registry_server_password": {
+      "type": "String"
+    },
+    "telemetry_opt_in": {
+      "type": "string",
+      "allowedValues": [
+        "Yes",
+        "No"
+      ]
+    },
+    "application_insights_instrumentation_key": {
```

Comment:
Can you update the documentation to include where to find this value in Azure portal?